test/scalability/mzcompose.py

```diff
@@ -307,12 +308,14 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
 
     endpoints: list[Endpoint] = []
     baseline_endpoint: Endpoint | None = None
-    for url, target in zip(args.materialize_url, args.target):
```

Comment:
`args.materialize_url` is only set for certain targets and `None` otherwise, causing issues with `zip`.

Reply:
Oops, sorry about that!

Reply:
No problem!